redis/Redis.php

```diff
@@ -2306,30 +2306,45 @@ public function brpoplpush( $srcKey, $dstKey, $timeout ) {}
     /**
      * Adds the specified member with a given score to the sorted set stored at key.
      *
-     * @param   string  $key    Required key
-     * @param   float   $score1 Required score
-     * @param   string  $value1 Required value
-     * @param   float   $score2 Optional score
-     * @param   string  $value2 Optional value
-     * @param   float   $scoreN Optional score
-     * @param   string  $valueN Optional value
+     * @param   string  $key     Required key
```

Comment:
Please, revert the formatting, so the only change in `@params` block is the added `@param array $options Options if needed`. We [do this to preserve git history](https://github.com/JetBrains/phpstorm-stubs/blob/master/CONTRIBUTING.md#code-style).